root/tree/index/runindex64.C

```diff
@@ -8,9 +8,11 @@ const char* fname = "index64.root";
 // Apple M1 has long double == double; these values exceed its range
 // and cannot be represented as (even temporary) expression results.
 // There would be a warning if you'd try.
-static constexpr bool shortlongdouble = sizeof(long double) < 16; // was true for __APPLE__ and __arm64__
-const Long64_t bigval   = shortlongdouble ? 0xFFFFFFFFFFFF : 0xFFFFFFFFFFFFFFF; // still positive number
+static constexpr bool shortlongdouble = sizeof(long double) < 16 || true; // was true for __APPLE__ and __arm64__
```

Comment:
Why the `|| true` ?

Reply:
because otherwise, the roottest fails, as it's platform dependent, and there is only one index64.ref to compare with.

Reply:
This change the nature of the next few lines and essentially remove their use and what they were testing.  For example see 9e3843d4bf50bc34e6e15dfe7c027f029417d6c0.

The length of long double is platform dependent and thus indeed the result/behavior might be (the next 2 lines were the way to deal with it).  We may have to switch from relying on the output to more direct comparison for the new code to handle properly the platform dependence.

Reply:
What do you think about adding a switch here: https://github.com/ferdymercury/roottest/blob/majorminor/root/tree/index/Makefile
that calls 'sed' to replace the platform-differences in runIndex, if on Mac?

The other think I can think of is to define a custom ErrorHandler to equalize the things printed.

Reply:
> What do you think about adding a switch here .. Makefile.

I'll rather avoid increasing the complexity of the Makefile unless absolutely necessary.  In addition the `Makefile` are deprecated and will eventually be replaced by `CMake/CTest` fragment.

> The other think I can think of is to define a custom ErrorHandler to equalize the things printed.

I am a bit confused.  What is the code that lead to the difference? What is the actual difference?

Reply:
Note that event 5 is not found on Linux.